content/2-3.md

```diff
@@ -67,58 +70,59 @@ Skills required for this module.
-* Demonstrate use of voice commands, keystrokes, and gestures provided by speech and mainstream technologies to navigate web pages through images. Explore advanced functionality that some tools provide, such as presenting all images in a list where users can select the image they are interested in. Examples of how people with disabilities interact with the Web are provided in [Stories of Web Users](https://www.w3.org/WAI/people-use-web/user-stories/).
-* Explain that images may not be available because of data restrictions or slow connections. Turn off images using common extensions or the settings screen in most browsers. Show and compare pages with and without text alternatives. Emphasize how essential information is often missed due to the lack of text alternatives.
-* Explain that text alternatives are one of the techniques used by search engines to determine what an image is about. Indicate that the more accurate an text alternative is, the better its image could rank.
+* Explain that the HTML attribute `alt` should contain concise and clear information about the image. State why it is not necessary to include the word "image" in the text alternative because the HTML element `img` is already announced by assistive technologies. Mention that decorative images should have empty ("") values in the `alt` attributes, or that they should be included in the web page using CSS instead, so that they are ignored by assistive technologies. Examples of how to use `alt` to code text alternatives are provided in techniques [H2: Combining adjacent image and text links for the same resource](https://www.w3.org/WAI/WCAG21/Techniques/html/H2) and [H37: Using alt attributes on `img` elements](https://www.w3.org/WAI/WCAG21/Techniques/html/H37).
+* Demonstrate other ways to provide text alternatives, such as the HTML attribute `title` and WAI-ARIA attributes `aria-label` and `aria-labelledby`. Explain that these may not be well supported by older browsers and assistive technologies. Examples of how to use `aria-label` and `aria-labelledby` to provide descriptions for images are provided in techniques [ARIA6: Using aria-label to provide labels for objects](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA6.html) and [ARIA10: Using aria-labelledby to provide a text alternative for non-text content](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA10.html).
+* Show different examples of informative images, including images of text, and decorative images. Explain how the same image can have different meaning in different web pages or in different parts of the content, and how that impacts the necessary text alternative. Descriptions of informative and decorative images are provided in the WAI tutorials on [Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/) and [Informative Images](https://www.w3.org/WAI/tutorials/images/informative/).
```

Comment:
If this is to show different examples of informative images, then decorative images would not be included in that.

Reply:
This tries to explain that the same image can have different meanings in different contexts, to the point that in some cases it may end up being decorative.

What about

> * Show different examples of informative images (including images of text) and contrast them with decorative images. Explain how the same image can have different meaning in different web pages or in different parts of the content. Emphasize how each meaning may require a different text alternative, including an empty text alternative when the image is decorative. Descriptions of informative and decorative images are provided in the WAI tutorials on [Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/) and [Informative Images](https://www.w3.org/WAI/tutorials/images/informative/).

Reply:
Ah, that makes sense. Some suggested edits to your suggest:

> Show different examples of informative images (including images of text) and contrast them with decorative images. Explain how context can effect the meaning of the same image. Emphasize how each context may require a different text alternative, including an empty text alternative when the image is decorative. Descriptions of informative and decorative images are provided in the WAI tutorials on [Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/) and [Informative Images](https://www.w3.org/WAI/tutorials/images/informative/).